javascript/builtins/String.json

```diff
@@ -1263,7 +1263,7 @@
                   "version_added": "29"
                 },
                 "firefox_android": {
-                  "version_added": false
+                  "version_added": "56"
```

Comment:
This one is the most remarkable. Did something special happen in Firefox 56? Or is the Firefox 29 detail here off the wall?